src/lib.rs

```diff
@@ -862,8 +862,23 @@ impl Default for Triangle {
 
 impl Triangle {
     ///
-    /// Creates a new `Triangle` given the data. This is done
-    /// to avoid boilerplate.
+    /// Creates a new `Triangle` given the necessary data.
+    ///
+    /// - The fields `a`, `b`, and `c` are the indices into
+    ///   [`BaseShape::initial_points()`] which are the vertices
+    ///   of this triangle.
+    ///   Note that this crate assumes
```

Comment:
I feel like this might be redundant given that it's stated already on line 829 for `Triangle`'s struct documentation. I would keep it here though.

Reply:
That's fair -- in theory it shouldn't matter anyway.